src/Domain/CoordinateMaps/CubicScale.hpp

```diff
@@ -28,66 +28,78 @@ namespace domain {
 namespace CoordMapsTimeDependent {
 /*!
  * \ingroup CoordMapsTimeDependentGroup
- * \brief CubicScale map defined by \f$x = a(t)*\xi+(b(t)-a(t))\xi^3/X^2\f$,
- *  where \f$X\f$ is the outer boundary.
+ * \brief Maps the radius \f$r\f$ as \f$r(t) = a(t)\rho + \left(b(t) - a(t)
+ * \frac{\rho^3} {R^2}\right)\f$ where \f$\rho\f$ is the radius of the source
+ * coordinates.
  *
- * The map scales the coordinates \f$\xi\f$ near the center by a factor
- * \f$a(t)\f$, while the coordinates near the outer boundary \f$X\f$, are scaled
+ * The map scales the radius \f$\rho\f$ in the source coordinates by a factor
+ * \f$a(t)\f$, while the coordinates near the outer boundary \f$R\f$, are scaled
  * by a factor \f$b(t)\f$. Here \f$a(t)\f$ and \f$b(t)\f$ are FunctionsOfTime.
  *
- * Currently, only a 1-D implementation.
+ * The inverse map is computed by solving the cubic equation:
+ *
+ * \f{align}{
+ * (b-a)\frac{\rho^3}{R^2} + a \rho - r = 0,
+ * \f}
+ *
+ * which is done by defining \f$q=\rho/R\f$, and solving
+ *
+ * \f{align}{
+ * q ((b-a) q^2 + a) - \frac{r}{R} = 0.
+ * \f}
+ *
+ * The source coordinates are obtained using:
```

Comment:
So I understand, why are the source coordinates not just $x^i$?

Reply:
Because the mapped coordinates are $x^i$. I'm not sure exactly what you're confused on, sorry. I use `\xi` for the source and `x` for the target.

Reply:
I wasn't clear that x^i was mapped and \xi^i was source. Now I get it. Maybe saying that in docs would help future people who don't get this right away?